sippy-ng/src/releases/ReleaseOverview.js

```diff
@@ -424,6 +424,56 @@ export default function ReleaseOverview(props) {
               </Card>
             </Grid>
 
+            <Grid item md={6} sm={12}>
+              <Card elevation={5} style={{ textAlign: 'center' }}>
+                <Typography
+                  component={Link}
+                  to={`/tests/${props.release}/details?${queryForBookmark(
+                    BOOKMARKS.RUN_7,
+                    BOOKMARKS.NO_NEVER_STABLE,
+                    BOOKMARKS.NO_TECHPREVIEW,
+                    BOOKMARKS.WITHOUT_OVERALL_JOB_RESULT,
+                    BOOKMARKS.NO_STEP_GRAPH,
+                    BOOKMARKS.HIGH_DELTA_FROM_AVERAGE,
+                    BOOKMARKS.HIGH_STANDARD_DEVIATION
+                  )}&sortField=delta_from_average&sort=asc`}
+                  style={{ textAlign: 'center' }}
+                  variant="h5"
+                >
+                  Top failing test NURPs
+                  <Tooltip title={TOP_FAILERS_TOOLTIP}>
+                    <InfoIcon />
```

Comment:
We could describe in the tooltip here that these are tests that are performing significantly worse on particular variant combos than the test in general.